stun_server/dns.tf

```diff
@@ -0,0 +1,24 @@
+resource "cloudflare_load_balancer" "stun-server" {
```

Comment:
This will not be the last service we want to geolocate from CF, lets do a module for CF LB, if that is too much at least do one for the lat/lon values

Reply:
Migrated to the module approach. Is this what you meant? Cause after writing it out, it seems even longer to me and does not make much sense 😅.

Reply:
After further clarification, I understand what you meant and did that. This makes much more sense now 👍🏻